src/components/transport_manager/include/transport_manager/transport_adapter/connection.h

```diff
@@ -64,6 +64,11 @@ class Connection {
    * @brief Disconnect the current connection.
    */
   virtual TransportAdapter::Error Disconnect() = 0;
+
+  /**
+   * @brief Terminate the current connection.
```

Comment:
@SNiukalov please provide more information or reference between  Disconnect and Terminate ?

Reply:
@alexkutsan 
Fixed in: https://github.com/smartdevicelink/sdl_core/pull/3134/commits/53e3d5496660cab96c95bc323d30cbfa23bb1895